docs/apis/subsystems/output.md

```diff
@@ -0,0 +1,281 @@
+---
+title: Output API
+tags:
+  - Output
+  - API
+---
+
+<!-- cspell:ignore renderables, sometext, courseid, enabletrusttext -->
+
+## Overview
```

Comment:
I'd recommend taking this out. We've standardised to remove the first title if it's something like "Introduction", or "Overview". There is no benefit.